tools/plugins/com.liferay.ide.server.core/src/com/liferay/ide/server/core/portal/AbstractPortalBundle.java

```diff
@@ -309,6 +318,38 @@ else if (configType.equals(_CONFIG_TYPE_SERVER)) {
 		return configInfoPath;
 	}
 
+	private TimeZone _getPortalDefaultTimeZone() {
+		IPath portalLibPath = getAppServerPortalDir().append("WEB-INF/lib");
```

Comment:
What about for servers that are running in a docker container and you don't have access to the appServerPortalDir

Reply:
i think docker we should not use same way because we can only use docker command can expose log content and we almost don't know what's type server. I feel docker sever should be a new  server type. Base on this I assume this will be only apply to normal 7.x jboss server